src/nsis.mk

```diff
@@ -17,15 +17,16 @@ define $(PKG)_UPDATE
 endef
 
 define $(PKG)_BUILD
+    $(if $(findstring x86_64-w64-mingw32,$(TARGET)),\
+        $(SED) -i 's/pei-i386/pei-x86-64/' '$(1)/SCons/Config/linker_script')
     cd '$(1)' && scons \
         MINGW_CROSS_PREFIX='$(TARGET)-' \
         PREFIX='$(PREFIX)/$(TARGET)' \
         `[ -d /usr/local/include ] && echo APPEND_CPPPATH=/usr/local/include` \
         `[ -d /usr/local/lib ]     && echo APPEND_LIBPATH=/usr/local/lib` \
-        SKIPUTILS='NSIS Menu' \
+        SKIPPLUGINS='System' \
+        SKIPUTILS='NSIS Menu,Makensisw,zip2exe' \
```

Comment:
This takes away features originally included in the 32-bit build. Is it _very_ difficult to get them working on 64-bit? If so, please only exclude them on 64-bit builds.